modules/instrument_manager/help/instrument_manager.md

```diff
@@ -1,54 +1,15 @@
 # Instrument manager
 
-## Uploading Instruments
+This module allows you to view the status of instruments that are installed in LORIS. If configured, the module will contain an **Upload** tab, where you can install new instruments created with the Instrument Builder. 
 
-The instrument manager provides the ability to view the status of
-instruments that are installed in LORIS and, if configured, the
-ability to install new instruments created with the instrument
-builder.
+## View Installed Instruments
```

Comment:
pls change to: `View Status of installed instruments`

Reply:
done!